MBBSEmu.Tests/ExportedModules/Majorbbs/genrdn_Tests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+using Xunit;
+
+namespace MBBSEmu.Tests.ExportedModules.Majorbbs
+{
+    public class genrdn_Tests : ExportedModuleTestBase
+    {
+        private const int GENRDN_ORDINAL = 315;
+
+        [Theory]
+        [InlineData(1, 3)]
```

Comment:
let's have another test where min == max (should return min) and min > max (should return min)